api/graphql/index.js

```diff
@@ -165,6 +165,12 @@ export function makeAuthenticatedQueries (userId, fetchOne, fetchMany) {
     post: (root, { id }) => fetchOne('Post', id),
     posts: (root, args) => fetchMany('Post', args),
     people: (root, args) => fetchMany('Person', args),
+    projects: (root, { context, contextId, viewerId }) => {
```

Comment:
In terms of the graphql modeling for this particular feature I'd prefer to see `projects` as a resolver on Person e.g. `person (id: $id) { projects { ... }  }`. There may be a case in the future for a root level 'projects' resolver, but I think for the Network and Community landing pages I would apply the same log (i.e. a `projects` resolver on Community and Network, etc). There is no hard and fast rule here, and graphql is a fairly open pattern, but the context for the projects query here is set by a query already going out for the person/member being viewed.

Reply:
i think i agree with Loren, but until we decide, this is ok

Reply:
i guess it depends on whether it makes sense to load the projects as part of 1 big query or whether it might be more efficient to load that section separately with a separate query. i dont have a strong opinion. Sticking with current norms we would load through the person resolver, though i think that query might be getting perhaps too large... All in all I would lean towards Loren's suggestion